README.md

```diff
@@ -51,7 +51,7 @@ You need to install prerequisites below to make Azure IoT Edge extension work:
 ## Prerequisites for specific language
 To develop IoT Edge modules you also need to install following prerequisites for specific language:
   #### C# module prerequisites
-  - [.NET Core 2.1 SDK or higher](https://www.microsoft.com/net/download)
+  - [.NET Core 2.1 SDK or higher](https://www.microsoft.com/net/download). [.NET Core 3.0 SDK or higher](https://dotnet.microsoft.com/download/dotnet-core/3.0) is required for ARM64 support.
```

Comment:
Add 3.0 part into  "()"?

Reply:
Shall we emphasize .NET Core 3.0 and ARM64 support is in preview? And it will be good to put a short link here saying "for more details about ARM64 (preview) support, please read the instruction here", and point it to a blog post Xin is gonna publish.

Reply:
I have updated the content, please help take a look